pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -126,11 +127,21 @@ func manageKubeSchedulerConfigMap_v311_00_to_latest(lister corev1listers.ConfigM
 	return resourceapply.ApplyConfigMap(client, recorder, requiredConfigMap)
 }
 
-func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, operatorConfig *operatorv1.KubeScheduler, imagePullSpec string) (*corev1.ConfigMap, bool, error) {
+func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, operatorConfig *operatorv1.KubeScheduler, imagePullSpec string, featureGateLister configlistersv1.FeaturesLister) (*corev1.ConfigMap, bool, error) {
 	required := resourceread.ReadPodV1OrDie(v311_00_assets.MustAsset("v3.11.0/kube-scheduler/pod.yaml"))
 	if len(imagePullSpec) > 0 {
 		required.Spec.Containers[0].Image = imagePullSpec
 	}
+	// check for feature gates from feature lister.
+	featureGates, err := checkForFeatureGates(featureGateLister)
+	if err != nil {
+		// Default behaviour
+		glog.Infof("Error while getting feature gates, so defaulting to empty feature gates")
```

Comment:
should log the error here.

Reply:
I prefer to either return error or log error but not both. I forgot to log it at the source, I did it there I think no need to do it here.